tests/integration_test/e2e_test.dart

```diff
@@ -38,9 +38,9 @@ void main() {
   group('FlutterFire', () {
     if (kIsWeb || !Platform.isWindows) {
       firebase_core.main();
+      firebase_auth.main();
       firebase_database.main();
       firebase_crashlytics.main();
-      firebase_auth.main();
```

Comment:
any reason for the change?

Reply:
We have add a race condition with auth for a bit of time now, and I tried this out of curiosity and seems to work better ... Might be just placebo tho 😅